modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1847,7 +1847,7 @@ int64_t CvCapture_FFMPEG::get_bitrate() const
 
 double CvCapture_FFMPEG::get_fps() const
 {
-#if 0 && LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(55, 1, 100) && LIBAVFORMAT_VERSION_MICRO >= 100
+#if LIBAVFORMAT_BUILD >= CALC_FFMPEG_VERSION(55, 1, 100) && LIBAVFORMAT_VERSION_MICRO >= 100
     double fps = r2d(av_guess_frame_rate(ic, ic->streams[video_stream], NULL));
```

Comment:
> `LIBAVFORMAT_VERSION_MICRO >= 100`

It is for FFmpeg only. What is about libav?

May be reimplement that function here?
https://github.com/FFmpeg/FFmpeg/blame/n3.4.1/libavformat/utils.c#L4936

---

It makes sense to target this patch for 3.4 branch too - there is libav on Linux builder.

Reply:
`av_guess_frame_rate` does not exist in libav. I tried libav repo master and several release tags.